src/custom/state/swap/actions.ts

```diff
@@ -0,0 +1,22 @@
+import { createAction } from '@reduxjs/toolkit'
+
+export enum Field {
+  INPUT = 'INPUT',
+  OUTPUT = 'OUTPUT',
+}
+
+// Mod: interface for replaceSwapState() action. Added chainId parameter, field renamed to independentField
+export interface ReplaceSwapStatePayload {
+  chainId: number | null
+  independentField: Field
+  typedValue: string
+  inputCurrencyId?: string
+  outputCurrencyId?: string
+  recipient: string | null
+}
+
+export const selectCurrency = createAction<{ field: Field; currencyId: string }>('swap/selectCurrency')
+export const switchCurrencies = createAction<void>('swap/switchCurrencies')
+export const typeInput = createAction<{ field: Field; typedValue: string }>('swap/typeInput')
+export const replaceSwapState = createAction<ReplaceSwapStatePayload>('swap/replaceSwapState')
+export const setRecipient = createAction<{ recipient: string | null }>('swap/setRecipient')
```

Comment:
This is not usually how we Mod the files. 

We would create `custom/state/swap/actions/index.ts` which re-export all but `replaceSwapState` (only difference). Then it defines and exports its own `replaceSwapState` no even need to mod anything here